src/pages/HomePage/HomePageComponents/Footer/Footer.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import "../../../../Colours/colours.css";
+import '../Footer/Footer.css'
+import { FaGithub,FaLinkedin,FaTwitter } from 'react-icons/fa';
```

Comment:
Nice. Didn't know this existed. Learnt something new.

Reply:
Thank You 🙂